packages/core-utils/src/browser-storage.ts

```diff
@@ -0,0 +1,174 @@
+import _ from 'lodash/fp';
```

Comment:
Moved this files to packages.
Changed imports to reference items within packages
Changed variables to have minimum necessary type (any)

Still includes import from lodash. Can replace functionality but wanted to keep PR's scope small.

Reply:
Different perspective on diff
![image](https://github.com/DataBiosphere/terra-ui/assets/24514630/68ae5974-2f8c-4516-b050-8be7aa7a9801)
![image](https://github.com/DataBiosphere/terra-ui/assets/24514630/91411211-cfc2-4994-881b-320d44e0d5af)
![image](https://github.com/DataBiosphere/terra-ui/assets/24514630/32cec86d-79de-4245-8da0-dcf19a625ed3)
![image](https://github.com/DataBiosphere/terra-ui/assets/24514630/b774e254-aae6-4fbf-9ce3-313543fd7e28)
![image](https://github.com/DataBiosphere/terra-ui/assets/24514630/fa4d16b3-14c0-4ccd-9ea7-0ed9c6d4d714)